testutil/keeper/sp.go

```diff
@@ -15,15 +15,18 @@ import (
 	"github.com/cosmos/cosmos-sdk/baseapp"
 	"github.com/cosmos/cosmos-sdk/codec"
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/runtime"
```

Comment:
Because we have new approach for test, so we do not need this file anymore.
`storage.go` should also be deleted.